app/provider/language/constants.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ *
+ * LanguageProvider constants
+ *
+ */
+
+export const CHANGE_LOCALE = 'app/LanguageToggle/CHANGE_LOCALE'
```

Comment:
Can you move `provider/language` to `app/language`. It's not specific to provider (provider is the one who runs the VPN node, hence _provides_ the service)?